crates/amaru/src/bin/amaru/cmd/daemon.rs

```diff
@@ -102,32 +101,10 @@ pub async fn run_pipeline(pipeline: gasket::daemon::Daemon, exit: CancellationTo
 }
 
 fn parse_args(args: Args) -> Result<Config, Box<dyn std::error::Error>> {
-    // TODO: Figure out from ledger + consensus store
-    let root = args.data_dir.join(args.network.to_string());
-
-    let nonces = read_csv(&root.join("nonces.csv"), nonce::from_csv)?;
```

Comment:
Shall the `nonces.csv` file be removed then?

Reply:
Yes, will do in an upcoming commit.